cpp/benchmarks/flow_simulation_ode_secirvvs.h

```diff
@@ -128,10 +127,8 @@ class FlowlessModel : public CompartmentalModel<osecirvvs::InfectionState,
             dydt[SIIi] = 0;
             dydt[EIIi] = 0;
 
-            double rateE =
-                1.0 / (2 * params.get<osecirvvs::SerialInterval>()[i] - params.get<osecirvvs::IncubationTime>()[i]);
-            double rateINS =
-                0.5 / (params.get<osecirvvs::IncubationTime>()[i] - params.get<osecirvvs::SerialInterval>()[i]);
+            double rateE   = 1.0 / params.get<osecirvvs::TimeExposed>()[i];
```

Comment:
Maybe also use directly, no need for a variable

Reply:
agree. should be put in equations directly as other rates.